src/module/data/actor-character.mjs

```diff
@@ -45,12 +56,37 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 						max: 4,
```

Comment:
This is from my original code, but we need to drop this back down to 3 as the max.